setup.py

```diff
@@ -16,20 +16,20 @@
     long_description = f.read()
 
 tests_require = [
-    'attrs>=17.4.0',
-    'beautifulsoup4>=4.6.0',
-    'Click>=7.0',
-    'flake8>=3.4.1',
-    'hypothesis>=6.0.0',
-    'mypy-extensions==0.4.1',
-    'py>=1.5.0',
-    'pylint>=1.7.2',
-    'pytest-cov',
-    'pytest-vcr>=1.0.2',
-    'pytest>=6.2.1',
-    'PyYAML>=4.2b1',
-    'setuptools>=41.0.1',
-    'vcrpy>=1.13.0',
+    "attrs>=17.4.0",
+    "beautifulsoup4>=4.6.0",
+    "Click>=7.0",
+    "flake8>=3.4.1",
+    "hypothesis>=6.0.0",
+    "mypy-extensions==0.4.1",
+    "py>=1.5.0",
+    "pylint>=1.7.2",
+    "pytest-cov",
+    "pytest-recording>=0.12.1",
+    "pytest>=6.2.1",
+    "PyYAML>=4.2b1",
+    "setuptools>=41.0.1",
+    "vcrpy>=1.13.0",
```

Comment:
…I thought the PR guidelines said to use single-quotes?

Reply:
at this point it is mixed between single and double quote at the repo. but i will use single to minimize the diff